src/config.py

```diff
@@ -1,4 +0,0 @@
-from dotenv import load_dotenv
-
-load_dotenv(".env.shared")
-load_dotenv(".env.secret")
```

Comment:
Is there documentation on what env vars a available?

Reply:
Well then you need to change the file to say that it specifies where to add the env-vars. (They are not loaded from the file anymore, correct?)

Reply:
They are still loaded from a file 😇 We're going to add a poe command to run the server. Then we'll specify `.env.shared` and `.env.secret` as env-files in the command 👍

Reply:
Is there an issue for that?

Reply:
see #99